mobile/src/main/resources/application.properties

```diff
@@ -78,12 +78,14 @@ tango.maxParticipants=300
 # for testing emails in applications. We will use it to verify if the submission works. Please
 # download the application and simply run it by invoking: java -jar fakeSMTP-<version>.jar.
 # After launching Fake SMTP Server, start this server, and you can read the emails generated there.
-email.host=localhost
-email.port=2525
+# Email Configuration
+# ========================
+email.host=smtp.gmail.com
+email.port=587
 email.protocol=smtp
-email.auth=false
-email.username=na
-email.password=na
+email.auth=true
+email.username=projectimplicitmentalhealth1@gmail.com
+email.password=4pQ%%Ur1pj
```

Comment:
Diheng, if you commit the password in an open source project like this then the whole world knows how to send emails from our project Implicit test account.  We need to pull this out and change the password.